File: Natural.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;



    public class Natural : Cliente
    {
        private string dni;
        private string nombres;
        private string apellidos;

        public Natural(int id_cliente, int telefono, string direccion, string cuentaBancaria,string dni, string nombres, string apellidos) : base(id_cliente,telefono,direccion,cuentaBancaria)
        {
            this.Dni = dni;
            this.Nombres = nombres;
            this.Apellidos = apellidos;
        }

        public string Dni{
		set{
			this.dni = value;
		}
		get{
			return dni;
		}
	    }
		
		public string Nombres{
		set{
			this.nombres = value;
		}
		get{
			return nombres;
		}
	    }
		
        public string Apellidos{
		set{
			this.apellidos = value;
		}
		get{
			return apellidos;
		}
	    }
    }

File: java/Natural.java
public class Natural extends Cliente{
	private String DNI;
	private String nombre;
	private String apellidos;

	public Natural(int id_cliente, int telefono, String direccion, String cuenta,
					String DNI, String nombre, String apellidos){
		super(id_cliente, telefono, direccion, cuenta);
		setDNI(DNI);
		setNombre(nombre);
		setApellidos(apellidos);
	}

	public String getDNI() {
        return DNI;
    }

    /**
     * @param DNI the DNI to set
     */
    public void setDNI(String DNI) {
        this.DNI = DNI;
    }

    /**
     * @return the nombre
     */
    public String getNombre() {
        return nombre;
    }

    /**
     * @param nombre the nombre to set
     */
    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    /**
     * @return the apellidos
     */
    public String getApellidos() {
        return apellidos;
    }

    /**
     * @param apellidos the apellidos to set
     */
    public void setApellidos(String apellidos) {
        this.apellidos = apellidos;
    }
}